### ISSUE TITLE
error in main.py

### ISSUE THREAD

**Issue by leowjs** [viriditass/Style-transfer#1]
For 'main.py' file, these rows    

## Style transfer setting
    style_layers = ['c11', 'c21', 'c31', 'c41', 'c51']
    content_layer = ['c42']

    # loss setting
    loss_layer = style_layers + content_layer
    fn_loss = [GramMSELoss()] * len(style_layers) + [nn.MSELoss()] * len(content_layer)
    if args.histogram_steps is not None:
      fn_loss += [HistogramLoss(num_steps=150, cuda=True)]
    fn_loss = [loss.to(device) for loss in fn_loss]

    if args.histogram_steps is not None:
        loss_layer = loss_layer + HistogramLoss(num_steps=args.histogram_steps, cuda=True)

I have error when executing this 
 loss_layer = loss_layer + HistogramLoss(num_steps=args.histogram_steps, cuda=True)

the loss_layer variable seem cannot add function, only add string

**Comment by seungu-ji** [comment]
I'm really sorry for replying late.
Are you still experiencing this issue?
